### PR TITLE
Create releases.md

### PR DESCRIPTION
**Pull request by hesterbruikman**
### Summary
File that captures release notes for App store ad Play store. Informs users of relevant additions, changes, fixes.

### CODE REVIEW

**Review by cammellos — COMMENTED** [review]
looks ok, the only thing is that we tend to use capital letters for `md` files top level,so woudl be 
`RELEASES.md`

File: releases.md [hunk 1]
```diff
@@ -0,0 +1,889 @@
+# Mobile Release Notes
+
+###### tags: `mobile` `core`
```
Comment (jakubgs) [inline releases.md]: This is probably just noise.

File: releases.md [hunk 2]
```diff
@@ -0,0 +1,889 @@
...
+- [ ] Comms timeframe agreed on (Mon/Tue ping when both submissions are in)
+- [ ] Privacy policy reviewed and updated
+- [ ] App translations for key features merged and tested
+- [ ] Countries to be excluded
+
+#### Before publishing
+- [ ] Translations for comms available
+- [ ] Promotional content ready
+- [ ] Update Draft with release on https://github.com/status-im/status-react/releases
+- [ ] Add release to DO store and [update site](https://notes.status.im/s/status-release-upload#) (Jakub - requires credentials to upload)
```
Comment (jakubgs) [inline releases.md]: This checklist shouldn't be here. I'd move it to a separate file. Like `doc/RELEASE_CHECKLIST.md` or something.

File: releases.md (hunk 1 above)
Comment (hesterbruikman) [reply releases.md]: thanks removing

File: releases.md (hunk 2 above)
Comment (hesterbruikman) [reply releases.md]: agreed, will move to a separate file

**Review by jakubgs — APPROVED** [review]

**Pull request merged**